tests/out/glsl/functions-webgl.main.Vertex.glsl

```diff
@@ -1,4 +1,5 @@
 #version 300 es
+#extension GL_EXT_gpu_shader5 : require
```

Comment:
Based on the [ext docs](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_gpu_shader5.txt), it requires OpenGL ES 3.1.

Reply:
You're right forgot to add the check on the version